src/app/pages/global-reach/map/schoolinfo-head.html

```diff
@@ -1,4 +1,6 @@
 <template args="model"></template>
     <div class="list-heading">{model.name}</div>
-    <div class="list-address">{model.physical_city}{model.physical_state_province === null ? '' : ', '+model.physical_state_province}</div>
+    <div class="list-address">
+            {model.physical_city === null ? '' : model.physical_city}{model.physical_city !== null && model.physical_state_province !== null ? ',' : ''}
```

Comment:
As before, this is a lot of logic to include in the view. It should be a function call. You can import `generateCityState` for this module, too.